### ISSUE TITLE
Link to key PRs and meeting notes

### ISSUE THREAD

**Issue by paulirish** [acdlite/react-fiber-architecture#5]
Would love if you could link/reference some of the core React teams's activity around fiber into this document. It'd make it easier to understand the actual implementation and current status.

You probably know this better, but I'm seeing the following items:

* https://github.com/reactjs/core-notes/blob/master/2016-06/june-23.md#update-on-fiber
* https://github.com/facebook/react/issues/6170
* https://github.com/facebook/react/pull/6690
* https://github.com/facebook/react/pulls?utf8=%E2%9C%93&q=is%3Apr%20%5Bfiber%5D

Cheers and thanks for putting this together!

**Comment by acdlite** [comment]
I agree these would be good resources to link to. I've placed links to similar resources inline throughout the document, but a dedicated section with links could be useful, too.

**Comment by kidwm** [comment]
And this one: [Fiber Principles: Contributing To Fiber](https://github.com/facebook/react/issues/7942)

**Comment by paulirish** [comment]
Also.. how about [Lin Clark - A Cartoon Intro to Fiber - React Conf 2017 - YouTube](https://www.youtube.com/watch?v=ZCuYPiUIONs) ?
